### PR TITLE
Update kubernetes-aks-engine-custom-vnet.md

### PR DESCRIPTION
**Pull request by JesusAlvarezTorres**
Adding address space considerations when creating a custom vnet for use in Kubernetes clusters.

### CODE REVIEW

**Comment by PRMerger7** [comment]
@JesusAlvarezTorres : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by JesusAlvarezTorres** [comment]
@walterov FYI, please take a look at this change if possible.

File: azure-stack/user/kubernetes-aks-engine-custom-vnet.md [hunk 1]
```diff
@@ -47,6 +47,12 @@ Create a new subnet in your virtual network. You will need to the get the subnet
     
 6. In the subnet blade, make a note of the address range and the virtual network CIDR Block, for example: `10.1.0.0 - 10.1.0.255 (256 addresses)` and `10.1.0.0/24`.
 
+## Considerations for selecting an address space
```
Comment (walterov) [inline azure-stack/user/kubernetes-aks-engine-custom-vnet.md:50]: Are you sure you are doing the change over the latest doc? I notice that the instruction above your section starts with number 6. in the currently published doc it starts with number 8. See here: https://docs.microsoft.com/en-us/azure-stack/user/kubernetes-aks-engine-custom-vnet?view=azs-2005#create-custom-virtual-network

Comment (JesusAlvarezTorres) [reply azure-stack/user/kubernetes-aks-engine-custom-vnet.md:50]: This is on top of the latest master, you can go to Files Changed > Preview. In the preview the numbering is correct. 
You can also see the raw file from the latest master, in the source the numbers shown are 1-6 but in the preview are 1-8. The images take the remaining numbers.

https://raw.githubusercontent.com/MicrosoftDocs/azure-stack-docs/master/azure-stack/user/kubernetes-aks-engine-custom-vnet.md

**Comment by mattbriggs** [comment]
@walterov #in-progress

**Review by mattbriggs — APPROVED** [review]

**Comment by mattbriggs** [comment]
#sign-off

**Comment by PRMerger20** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**